test/unit/models/forms/v2/CorrespondenceDetailsFormSpec.scala

```diff
@@ -82,7 +82,7 @@ class CorrespondenceDetailsFormSpec extends ModelsBaseSpec {
         val form = CorrespondenceDetailsForm.correspondenceDetailsForm(correspondenceCase)
 
         form.hasErrors shouldBe false
-        form.data      shouldBe params.mapValues(v => v.head)
+        form.data      shouldBe params.map { case (fst, snd) => fst -> snd.head }
```

Comment:
Same with the others as it can just be `params.view.mapValues(v => v.head).toMap`. Thanks